### PR TITLE
[UI] Fix Visibility of Dark/Light Mode Toggle and Signup Button in Mobile View

### LINKED ISSUE #203: Dark/Light Mode and Signup Button Not Visible in Mobile View
#### Current Behavior
In the desktop view, the dark/light mode toggle and the signup button are visible. However, when switching to the mobile view, these options disappear. Upon inspecting the code, it appears that the display for these elements is set to none in the mobile view.

#### Expected Behavior
The dark/light mode toggle and signup button should be visible in both desktop and mobile views.

#### Screenshots/Logs
Mobile View(Actual):
image
image

Mobile View(Expected):
image
image
image

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by akshansh-modi**
**Notes for Reviewers**

This PR fixes #203 
It addresses the issue where the dark/light mode toggle and signup button were not visible in mobile view. The display property for these elements was set to none in mobile view. This PR ensures that both the dark/light mode toggle and signup button are now visible in both desktop and mobile views.

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: site/src/components/Navigation/Navigation.styles.js [hunk 1]
```diff
@@ -250,17 +250,14 @@ export const Header = styled.header`
     }
     .themeToggle {
       margin: 0 5px;
-      background: ${({ theme }) => theme.body};
```
Comment (Ashparshp) [inline site/src/components/Navigation/Navigation.styles.js:253]: Why were changes made beyond removing `.btn-container`? Removing it alone should resolve the issue, right?

Comment (akshansh-modi) [reply site/src/components/Navigation/Navigation.styles.js:253]: I made two additional changes:

1)Removing the background:
When switching the background, adding background: ${({ theme }) => theme.body}; was affecting the background of our SVGs unnecessarily.

2)Removing the border:
This adjustment was made to maintain consistency with the desktop view, where borders were not present, ensuring a cleaner appearance.

Comment (akshansh-modi) [reply site/src/components/Navigation/Navigation.styles.js:253]: and in desktop view also there is no switching in background of svg

Comment (akshansh-modi) [reply site/src/components/Navigation/Navigation.styles.js:253]: with 
  background: ${({ theme }) => theme.body};

https://github.com/meshery/play/assets/92106389/c68a6b83-5141-47b1-b566-1e6fdab1ae98

without :

https://github.com/meshery/play/assets/92106389/3ced7896-4a74-43e8-bcd4-f86cf5c5e302



with border 
image
without border 
image

Comment (Ashparshp) [reply site/src/components/Navigation/Navigation.styles.js:253]: @akshansh-modi thanks for the TED Talk on theme toggles and SVG backgrounds! 😄 Appreciate the insights and the videos, though next time, let's aim for a 30-second summary, shall we? Thanks for your contributions!

Comment (akshansh-modi) [reply site/src/components/Navigation/Navigation.styles.js:253]: 😄 Got it, I'll keep it concise next time.

**Comment by akshansh-modi** [comment]
@Ashparshp please accept the pull request so it can get merged, or there are some changes that I need to make?

**Review by Ashparshp — APPROVED** [review]
LGTM!

**Review by Ashparshp — APPROVED** [review]

**Comment by akshansh-modi** [comment]
Hi @Ashparshp, I noticed the PR has been approved but hasn't been merged yet. Is there anything else that needs to be addressed or any additional steps I can take to help move it forward? Appreciate your guidance on this

**Review by Yashsharma1911 — APPROVED** [review]
LGTM

**Comment by Yashsharma1911** [comment]
Thankyou @akshansh-modi for fixing this

**Pull request merged**